docker-compose.yml

```diff
@@ -0,0 +1,23 @@
+version: '2'
+services:
+  calypso:
+    image: node:6.9.4
+    extra_hosts:
+      - 'calypso.localhost:0.0.0.0'
+    environment:
+      CALYPSO_ENV: 'development'
+      NODE_ENV: 'development'
+      HOST: 'calypso.localhost'
+      PORT: 3000
+    ports:
+      - '3000:3000'
+    volumes:
+      - ./:/calypso
+      - node_modules_volume:/calypso/node_modules
+      - hooks_volume:/calypso/.git/hooks
```

Comment:
Apparently, this makes it so that it runs in windows. Linux symlinks do not play nice there...